tests/test_model.py

```diff
@@ -30,3 +33,14 @@ def test_model_should_predict_valid_shapes_at_inference(
     assert scores.shape == (n_images, yolo_max_boxes)
     assert classes.shape == (n_images, yolo_max_boxes)
     assert valid_detections.shape == tuple([n_images])
+
+
+@pytest.mark.parametrize("input_shape", [(32, 33, 3), (33, 32, 3)])
+def test_model_instanciation_should_fail_with_input_shapes_not_multiple_of_32(
+    input_shape
+):
+    with pytest.raises(ValueError):
+        YOLOv4(input_shape, 80, [])
+
+    with pytest.raises(ValueError):
+        YOLOv4(input_shape, 80, [])
```

Comment:
Duplicate code :) I think you forgot to remove one of the two use cases when you factorized the tests

Reply:
That's it! My bad :)